app/lib/getPharmacies.js

```diff
@@ -41,7 +47,41 @@ function nearby(searchPoint, geo, limit) {
   const sortedOrgs = nearbyOrgs.sort(sortByDistance);
   console.timeEnd('sort-nearby-orgs');
 
-  return sortedOrgs.slice(0, maxResults);
+  const numberOfOpenToReturn = 3;
+  let numberReturned = 0;
+
+  console.time('filter-open-orgs');
+  const openServices = sortedOrgs.filter((item) => {
+    // TODO: Look at breaking out of the loop once three have been found
+    const openingTimes = item.openingTimes;
+    let isOpen;
+    let openingTimesMessage;
+
+    if (openingTimes) {
+      const openingTimesMoment = new OpeningTimes(item.openingTimes.general, 'Europe/London');
+      openingTimesMessage = openingTimesMoment.getOpeningHoursMessage(moment());
+      isOpen = openingTimesMoment.isOpen(moment());
+    } else {
+      openingTimesMessage = 'Call for opening times';
+      isOpen = false;
+    }
+    /* eslint-disable no-param-reassign */
+    item.openingTimesMessage = openingTimesMessage;
```

Comment:
I had imagined that the openingTimesMoment would be assigned to item for use by the view (e.g. item.openingTimesMoment = openingTimesMoment).